modules/java/src/main/java/org/syslog_ng/SyslogNgClassLoader.java

```diff
@@ -146,18 +146,27 @@ private static int javaVersion() {
     return Integer.parseInt(versionStr[0]);
   }
 
+  private void expandClassPath_java9(URL[] urls) throws Exception {
+    InternalMessageSender.debug("Extend classpath according to java version 9");
+    classLoader = URLClassLoader.newInstance(urls ,classLoader);
```

Comment:
The space and comma is swapped: `urls ,classLoader` vs `urls, classLoader`.

Reply:
fixed.